client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-content/index.tsx

```diff
@@ -24,6 +24,7 @@ interface Props {
 
 export default function SiteContent( { data, isLoading, currentPage, isFavoritesTab }: Props ) {
 	const isMobile = useMobileBreakpoint();
+	const isDesktop = useDesktopBreakpoint();
```

Comment:
How about we can use the `isMobile` condition to display the mobile view and fall back to a large screen view if not mobile?

Reply:
`isMobile` is true for screens smaller than **480px**, while `isDesktop` is true for screens bigger than **960px**.